src/locales/de/translation.json

```diff
@@ -129,10 +129,6 @@
     "switchToMainnet": "Wechsel zu Mainnet."
   },
   "footer": {
-    "poweredBy": {
-      "oasismonitor": "Powered by Oasis Monitor API & Oasis gRPC",
-      "oasisscan": "Powered by Oasis Scan API & Oasis gRPC"
-    },
```

Comment:
Not sure this change belongs to this PR.

Reply:
Agreed. Was this an unintentional change?